contracts/QuestFactory.sol

```diff
@@ -143,10 +140,6 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         _;
     }
 
-    function mintReceipt(string memory, bytes32, bytes memory) external pure{
-        revert Deprecated();
-    }
```

Comment:
AFAIK if you push a call to a contract with a selector that isn't implemented it'll trigger the fallback so that behavior would make sense. Anyone with a stale ABI would be triggering a selector and then the fallback here doesn't revert.
![image](https://github.com/rabbitholegg/quest-protocol/assets/6549550/82f51514-6057-4548-bdfc-212f470a23ab)

We could add a revert into the `fallback` as an alternative or I can add this function back in. I'm neutral, but that explains the behavior you were observing.